.github/tests/bracket_test.py

```diff
@@ -0,0 +1,49 @@
+import re
+
+
+def check_brackets(filename):
+    """
+    :param filename: The name of the file to be checked for balanced brackets.
+    :return: None
+
+    This method checks if the brackets in a given file are balanced. It reads through each line of the file, strips any enumerations and smileys, and then checks if the brackets are balanced.
+
+    If a line contains an extra closing bracket that is not matched with an opening bracket, it will print a message indicating the line number and the issue.
+
+    If a line has an open bracket that is not closed, it will also print a message indicating the line number and the issue.
+
+    Note: The method does not return any value. It directly prints the error messages.
+
+    Example usage:
+    check_brackets("example.txt")
+    """
+    with open(filename, 'r', encoding="UTF-8-SIG") as file:
+        for line_number, line in enumerate(file, start=1):
+            if line_number != 44749 and line_number != 44750 and line_number != 46617 and line_number != 46654:
```

Comment:
Könnten wir das hier mit not in liste ersetzen?  Dann können wir schnell neue Ausnahme Zeilen hinzufügen

Reply:
Die Zeilennummern zu verwenden ist eh quatsch weil sich das nach jedem Patch ändern kann. Im 3.23 Evocati Build hat die Zeile 44749 eine neue Position 44034